dbus-macros/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "dbus-macros"
```

Comment:
The name "dbus-macros" is taken on crates.io, you need to have a different name.

Reply:
Maybe dbus-derive, that's not taken yet?

Reply:
Sounds good to me, double checked - still not taken